pennylane_cirq/simulator_device.py

```diff
@@ -177,8 +182,76 @@ def generate_samples(self):
         # Bring measurements to a more managable form, but keep True/False as values for now
         # They will be changed in the measurement routines where the observable is available
         return np.array(
-            [
-                self._result.measurements[str(wire)].flatten()
-                for wire in range(self.num_wires)
-            ]
+            [self._result.measurements[str(wire)].flatten() for wire in range(self.num_wires)]
         ).T.astype(int)
+
+
+class MixedStateSimulatorDevice(SimulatorDevice):
+    r"""Cirq mixed-state simulator device for PennyLane.
+
+    Args:
+        wires (int): the number of modes to initialize the device in
+        shots (int): Number of circuit evaluations/random samples used
```

Comment:
Nitpick: I think it would be nice to follow one convention here; i.e. either start with `the number of` or `Number of` (or `number of` or `The number of`). Don't really have a preference though. This is the case in several docstrings here.

Reply:
We do have a convention :smiley: :
- don't punctuate (capital/period)
*unless* the description starts becoming more than one sentence, then do turn it into proper sentences